code/addons/vitest/src/plugin/transformer.ts

```diff
@@ -1,13 +1,83 @@
+/* eslint-disable local-rules/no-uncategorized-errors */
+
+/* eslint-disable no-underscore-dangle */
 import { getStoryTitle } from 'storybook/internal/common';
+import { formatCsf, loadCsf } from 'storybook/internal/csf-tools';
 import type { StoriesEntry } from 'storybook/internal/types';
 
-import MagicString from 'magic-string';
-import { dedent } from 'ts-dedent';
-import typescript from 'typescript';
+import * as t from '@babel/types';
 
 import type { InternalOptions } from './types';
 
-// Main transform function for the Vitest plugin
+function getUtilityImports() {
+  return [
+    t.importDeclaration(
+      [t.importSpecifier(t.identifier('__test'), t.identifier('test'))],
```

Comment:
You can use `path.scope.generateUidIdentifier("test")` for this in babel.

https://github.com/jamiebuilds/babel-handbook/blob/master/translations/en/plugin-handbook.md#generating-a-uid

Reply:
but we don't have `path`?